examples/common.js

```diff
@@ -221,47 +143,10 @@
             }
         });
         this.getNamedLayers = function() {
-            return this.map.getLayers().getArray().filter(function(lyr) {
+            return this.storyMap.getStoryLayers().getArray().filter(function(lyr) {
```

Comment:
Okay see the first part of this change: https://github.com/bartvde/story-tools/commit/ecb1992af445027729af345c7b26cfb7f8146eff